debug.gemspec

```diff
@@ -29,4 +29,5 @@ Gem::Specification.new do |spec|
 
   spec.add_dependency "irb", ">= 1.5.0" # for binding.irb(show_code: false)
   spec.add_dependency "reline", ">= 0.3.1"
+  spec.add_development_dependency "zeitwerk", "~> 2.6.9"
```

Comment:
Imma be real, I have no idea what the difference is between this, and `gemspec.add_development_dependency` 🫥

Reply:
I think it just depends on how the variable from `Gem::Specification.new do |spec|` is named. In most cases it seems to be named `spec`?